src/additional/maxfiles.js

```diff
@@ -0,0 +1,16 @@
+//Limit the number of files in a FileList.
+$.validator.addMethod( "maxfiles", function( value, element, param ) {
+	if ( this.optional( element ) ) {
+		return true;
+	}
+
+	if ( $( element ).attr( "type" ) === "file" ) {
+		if ( element.files && element.files.length ) {
+			if ( element.files.length > param ) {
```

Comment:
Can you please merge the above two `if` blocks?

Reply:
Which two?  There are three of them.  Lines 8 and 9, I presume?

Reply:
Yes, 8 and 9.